_test/test_unique_objects_container/thingy.m

```diff
@@ -0,0 +1,32 @@
+classdef thingy
```

Comment:
I don't think `thingy` is a useful name, even `data_container` would be more informative

Reply:
more informative on usage, yes, but less informative on being a local idiomatic quirk to couple the test to the construct.